index.d.ts

```diff
@@ -0,0 +1,110 @@
+// Type definitions for express-session 1.16
+// Project: https://github.com/expressjs/session
+// Definitions by: Hiroki Horiuchi <https://github.com/horiuchi>
+//                 Jacob Bogers <https://github.com/jacobbogers>
+//                 Naoto Yokoyama <https://github.com/builtinnya>
+//                 Ryan Cannon <https://github.com/ry7n>
+//                 HoldYourWaffle <https://github.com/HoldYourWaffle>
+
+import express from 'express'
+import node from 'events'
+
+
+
+declare global {
+  namespace Express {
+    interface Request { //inject additional properties on express.Request
+      session: Session;
+      sessionID: string;
+    }
+  }
+
+  export interface SessionData {
+    //seperate interface to allow application-specific declaration merging (see README)
+  }
+}
+
+
+
+export default function session(options?: SessionOptions): express.RequestHandler;
+
+
+interface CookieOptions {
+  maxAge?: number;
+  signed?: boolean;
+  expires?: Date;
+  httpOnly?: boolean;
+  path?: string;
+  domain?: string;
+  secure?: boolean | 'auto';
+  sameSite?: boolean | string;
+}
+
+export class Cookie implements CookieOptions { //XXX actually a class
+  originalMaxAge: number;
+
+  maxAge?: number;
+  signed?: boolean;
+  expires?: Date;
+  httpOnly?: boolean;
+  path?: string;
+  domain?: string;
+  secure?: boolean | 'auto';
+  encode?: (val: string) => void;
```

Comment:
I don't think there is such a property as this available.

Reply:
I blame VS Code for auto-inserting this (╯°□°）╯︵ ┻━┻

Reply:
Fixed